lib/generators/shopify_app/app_proxy_controller/templates/app_proxy_controller.rb

```diff
@@ -0,0 +1,12 @@
+class AppProxyController < ApplicationController
+  if Rails.env.development? || Rails.env.test?
+   around_action :shopify_session
+  else
+   include ShopifyApp::AppProxyVerification
+  end
```

Comment:
Ohh.. now, why didn't I think of this..? lol! Thanks. 
I'll make a note of it in the README changes as well.. It'll be helpful to somebody out there, if this PR does get merged.
:)